plugin/ochttp/stats_transport.go

```diff
@@ -79,12 +79,44 @@ func (t statsTransport) CancelRequest(req *http.Request) {
 	}
 }
 
+// StatsHandler is an http.Handler that updates OpenCensus HTTP stats for
+// requests served by a net/http server.
+type StatsHandler struct {
```

Comment:
For gRPC we moved from having a separate stats and tracing handler to having a combined one (with the ability to opt-out of stats or tracing using booleans). We reasoned that in almost all cases, there's no reason not to configure both stats and tracing. The reason both stats and tracing are part of OpenCensus is that there are some really cool  things you can do when you have both, like exemplars: https://github.com/census-instrumentation/opencensus-specs/issues/46 - so we're kind of pushing people to use both together. What do you think?

Reply:
My contrary opinion was: I don't understand why you would coupled them :-)

The http.Handler interface means that if you want both pieces of functionality it is trivial to wrap one in the other. But if you're incentivising use by opting users into all functionality as standard, it makes sense to have everything in one place. I'll ditch the separate handler.

I didn't want to move the stats functionality into trace.go because that seemed counter-intuitive, so instead I've done a minor bit of refactoring. Hope that's ok?